applications/nrf_desktop/configuration/nrf52_pca20037/led_state_def.h

```diff
@@ -26,12 +26,27 @@ static const u8_t led_map[LED_ID_COUNT] = {
 /* System state LED is unavailable - leave undefined */
 static const struct led_effect led_system_state_effect[LED_SYSTEM_STATE_COUNT];
 
-static const struct led_effect led_peer_state_effect[CONFIG_BT_MAX_PAIRED][LED_PEER_STATE_COUNT] = {
+static const struct led_effect led_peer_state_effect[CONFIG_BT_MAX_PAIRED - 1][LED_PEER_STATE_COUNT] = {
```

Comment:
maybe we can introduce a define (or a config in kconfig) that would give peer count?

Reply:
Added in led_state.h using as defined value